lib/bundle.js

```diff
@@ -175,7 +176,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
     });
   }
 
-  return bundle()
+  return bundle(systemBuilder)
```

Comment:
This doesn't work because a few lines below there is a reference to `systemBuilder`. What I've done is move the variable down to be with this line so it's clearer. Done this in both places, let me know what you think.